src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+
+  static addAnimal(animal) {
+    this.alive.push(animal);
+  }
```

Comment:
just push it right away at the Animal constructor. So you won't need to do it in the inherited classes